foca/api/register_openapi.py

```diff
@@ -39,7 +39,16 @@ def register_openapi(
     # Iterate over OpenAPI specs
     for spec in specs:
         spec_modified = False
-        spec_parsed = ConfigParser.parse_yaml(spec.path)
+        spec_parsed = dict()
+        if isinstance(spec.path, list):
+            # if the list contains a single path, treat it as str
+            if len(spec.path) == 1:
+                spec.path = str(spec.path[0])
```

Comment:
Seems like in this case the specs are never parsed. Actually, if the user provides just a file path or a list with a single file path, the behavior should be the same: the file path should be parsed with `ConfigParser.parse_yaml()`.

Therefore, you can simplify the code, e.g., like this:

```py
if isinstance(spec.path, list) and len(spec.path) > 1:
    spec_modified = True
    spec_parsed = _mergeSpecs(*spec.path)
else:
    spec_parsed = ConfigParser.parse_yaml(spec.path)
```